src/back/project/src/model/cc_type_param.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+namespace svetit::project::model {
+
+struct CcTypeParam {
+	int ccTypeId;
+	int paramId;
+	bool isDeleted;
```

Comment:
Добавь пожалуйста значение по умолчанию везде.
```suggestion
	bool isDeleted = false;
```

Reply:
Готово.